src/components/common/UserInfoForm.vue

```diff
@@ -134,7 +135,15 @@ export default {
       weightUnit,
       v$,
       checkForm,
+      rules,
     } = useValidate();
+    const validator = useVuelidate(rules, {
+      name,
+    });
+    const avatarLetter = computed(() => {
```

Comment:
technicznie to nie jest avatarLetter tylko całe imię, więc albo zmienic tu na firstName np albo tutaj zwracac litere a nie w Avatar.vue